docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -10,77 +11,85 @@ This tutorial demonstrates how to extend the plug-in that is bundled with this s
-    }
+        public static readonly TODO_URI = "/todos";
+
+        public static getTodos(session: AbstractSession): Promise<ITodo[]> {
+            Logger.getAppLogger().trace("Typicode.getTodos() called");
+            return RestClient.getExpectJSON<ITodo[]>(session, Typicode.TODO_URI);
+        }
 
-    public static getTodo(session: AbstractSession, id: number): Promise<ITodo> {
-        Logger.getAppLogger().trace("Typicode.getTodos() called with id " + id);
-        ImperativeExpect.toNotBeNullOrUndefined(id, "id must be provided");
-        const resource = Typicode.TODO_URI + "/" + id;
-        return RestClient.getExpectJSON<ITodo>(session, resource);
+        public static getTodo(session: AbstractSession, id: number): Promise<ITodo> {
+            Logger.getAppLogger().trace("Typicode.getTodos() called with id " + id);
+            ImperativeExpect.toNotBeNullOrUndefined(id, "id must be provided");
+            const resource = Typicode.TODO_URI + "/" + id;
+            return RestClient.getExpectJSON<ITodo>(session, resource);
+        }
     }
-}
 
-```
+    ```
 
-The `Typicode` class provides two programmatic APIs, `getTodos` and `getTodo`, to get an array of `ITodo` objects or a specific
-`ITodo` respectively. The Node.js APIs use `@zowe/imperative` infrastructure to provide logging, parameter validation,
-and to call a REST API. See the [Imperative CLI Framework documentation](https://github.com/zowe/imperative/wiki) for more information.
+    The `Typicode` class provides two programmatic APIs, `getTodos` and `getTodo`, to get an array of `ITodo` objects or a specific `ITodo`, respectively.
+
+    The Node.js APIs use `@zowe/imperative` infrastructure to provide logging, parameter validation, and to call a REST API. See the [Imperative CLI Framework documentation](https://github.com/zowe/imperative/wiki) for more information. **[this repo is archived. what else can we link to here?]**
 
 #### Exporting interface and programmatic API for other Node.js applications
+**[Feels like there's a missing word in this ^^ header. Can we say "the" interface and programmatic ...?]**
 
-Update [zowe-cli-sample-plugin/src/index.ts](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/index.ts) to contain the following:
+Edit the [zowe-cli-sample-plugin/src/index.ts](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/index.ts) file to contain the following code: **[do we need to include the path in the file name?]** **[is "code" correct here?]**
 
 ```typescript
 export * from "./api/doc/ITodo";
 export * from "./api/Typicode";
 ```
 
-A sample invocation of your API might look similar to the following, if it were used by a separate, standalone Node.js application:
+A sample invocation of your API might look similar to the following, if it were used by a separate, standalone Node.js application: **[what does "invocation" mean here? can we use another, more clear way of describing what happens?]**
```

Comment:
@adam-wolfe please respond

Reply:
We could say something like this (the wording's not great):
"Here is an example of how a separate, standalone Node.js application could use APIs from the sample Typicode plug-in to get data from the REST API at jsonplaceholder.typicode.com:"

The point being that other Node.js applications can use functionality from your plug-in that you `export` as shown in the step above.